packages/glimmer-runtime/lib/builder.ts

```diff
@@ -89,18 +91,31 @@ class GroupedElementOperations implements ElementOperations {
   }
 
   addAttribute(name: InternedString, reference: PathReference<string>) {
-    if (this.element.tagName === 'INPUT' && name === 'value') {
-      this.group.push(new EmptyStringResetingProperty(this.element, name, reference));
+    let { element } = this;
+    let { tagName } = element;
+
+    if (SanitizedNonNamespacedAttribute.requiresSanitization(tagName, name)) {
+      this.group.push(new SanitizedNonNamespacedAttribute(element, name, reference));
+    } else if (tagName === 'INPUT' && name === 'value') {
+      this.group.push(new EmptyStringResetingProperty(element, name, reference));
+    } else {
+      this.group.push(new NonNamespacedAttribute(element, name, reference));
     }
```

Comment:
Interesting, I would have thought the tests would fail if you did this but apparently not. I had assumed you needed both. I added a specific test case around this though, so I'm not too worried about this change breaking things.